pages/manual/planet.md

```diff
@@ -533,7 +533,7 @@ As you can see below, once you are in your planet dashboard you can access the p
 ![Access Meetup](images/planet-access-meetup.png)
 
 ### Available Actions On Meetup Page
-1. Traversing the page. Here you can **filter and search** (Red box) for specific meetups and use the 
+1. Traversing the page. Here you can **filter and search** (Red box) for specific meetups and use the
```

Comment:
As you are removing extra space its better to not break this line as it is single sentence

Reply:
I think this has been taken care of with a new commit to cap_error